java/build.gradle.kts

```diff
@@ -24,8 +24,8 @@ spotless {
 }
 
 dependencies {
-    compileOnly(platform("io.opentelemetry:opentelemetry-bom:1.21.0"))
-    compileOnly(platform("io.opentelemetry:opentelemetry-bom-alpha:1.21.0-alpha"))
+    compileOnly(platform("io.opentelemetry:opentelemetry-bom:1.23.0"))
```

Comment:
should we use 
https://search.maven.org/artifact/io.opentelemetry/opentelemetry-bom/1.22.0/pom